_layouts/documentation.html

```diff
@@ -56,6 +59,18 @@ <h1>{{page.title}}</h1>
         </ul>
 
         {% include documentation-sections.html sections=page.scala2-sections %}
+      </div>
+    </div>
+
+    <div class="wrap scala3">
+      <div class="language-header">
+        <h1>Scala 3</h1>
+      </div>
+      <div class="inner-box">
+        <blockquote>
+          We are still in the process of writing the documentation for Scala 3.
+          You can <a href="/scala3/contribute-to-docs.html">help us to improve the documentation</a>.
+        </blockquote>
         {% include documentation-sections.html sections=page.scala3-sections %}
       </div>
     </div>
```

Comment:
Is this `documentation` layout used now?

Reply:
it is used for most of the translated index pages